README.md

```diff
@@ -51,10 +51,15 @@ end
 - path - the path, where your EmberCLI applications is located. The default
   value is the name of your app in the Rails root.
 
+- enable - a lambda that accepts each requests' path. The default value is a
+  lambda that returns `true`.
+
 ```ruby
 EmberCLI.configure do |c|
   c.app :adminpanel # path is "<your-rails-root>/adminpanel"
-  c.app :frontend, path: "/path/to/your/ember-cli-app/on/disk"
+  c.app :frontend,
+    path: "/path/to/your/ember-cli-app/on/disk",
+    enable: -> path { path.starts_with?("/app/") }
```

Comment:
@rwz I've covered the positive use case, would it make sense also covering a negative case (`!path.starts_with?("/api/")`)?